<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SM3widget</class>
 <widget class="QWidget" name="SM3widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>470</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>550</width>
    <height>470</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>550</width>
    <height>470</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>文件加密保护</string>
  </property>
  <widget class="QLineEdit" name="filelineEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="filepushButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>60</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>选择文件</string>
   </property>
  </widget>
  <widget class="QPushButton" name="hashpushButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>120</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>计算文件hash值</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>230</y>
     <width>421</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="backpushButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>420</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>190</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>SM3文件hash</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
